lib/kdump_utils.pm

```diff
@@ -96,6 +96,13 @@ sub activate_kdump {
     my @tags = qw(yast2-kdump-disabled yast2-kdump-enabled yast2-kdump-restart-info yast2-missing_package yast2_console-finished);
     do {
         assert_screen \@tags, 300;
+        # for ppc64le and aarch64 we need increase kdump memory, see bsc#957053 and bsc#1120566
+        if (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'aarch64')) {
+            wait_screen_change { send_key ' alt-y' };
+            type_string '640';
+            send_key 'ret';
+            record_soft_failure 'increase kdump memory size or kdumptool gets killed by OOM, bsc#957053 and bsc#1120566';
```

Comment:
We should adjust this reference to bugs. I am not yet sure how openqa-review handles this text but I would suggest to not annoy the recipients of bsc#957053 which is closed with references to this test especially as we still do not check the error condition.

Reply:
> We should adjust this reference to bugs. I am not yet sure how openqa-review handles this text but I would suggest to not annoy the recipients of bsc#957053 which is closed with references to this test especially as we still do not check the error condition.

addressed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6725